_data/projects/saleor.yml

```diff
@@ -0,0 +1,18 @@
+name: Saleor
+desc: Saleor is a high-performance, GraphQL-first e-commerce solution created with Python and Django. https://getsaleor.com/
+site: https://github.com/mirumee/saleor/
+tags:
+- Python
+- Django
+- GraphQL
+- React
+- e-commerce
+- GraphQL
```

Comment:
@Pacu2 This tag is being used twice due to which travis is failing.

Reply:
Right, good catch, thanks